staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -428,9 +429,9 @@ func TestNoRestraint(t *testing.T) {
 			newUniformClient(1001001001, 5, 10, time.Second, time.Second),
 			newUniformClient(2002002002, 2, 10, time.Second, time.Second/2),
 		},
-		concurrencyLimit:       10,
+		concurrencyLimit:       2,
 		evalDuration:           time.Second * 15,
-		expectedFair:           []bool{true},
+		expectedFair:           []bool{false},
```

Comment:
To double check - this was simply a bug that it was marked as fair, right?

Reply:
Not exactly.  With the larger concurrency limit, the NoRestraint implementation yields the fair result.  The real bug was not testing with a small concurrency limit.

Reply:
"bug" is perhaps too strong a word here.  I think it is more interesting to test that the NoRestraint queueset is unfair when there is competition than it is to show that the fair result is delivered in the absence of competition.  However, there is some value in the latter --- it gives some degree of validation to test harness and its evaluation method.